src/daily_output.ts

```diff
@@ -0,0 +1,432 @@
+import { readFileSync, writeFileSync } from 'fs';
+import { GROUP_EFFORT_REWARDS, ONE_THOUSAND_MB, OUTPUT_NAME, OUTPUT_TEMPLATE_NAME } from './constants';
+import { GQLEdgeInterface } from './gql_types';
+import { OutputData, StakedPSTHolders, WalletsStats, WalletStatEntry } from './inferno_types';
+
+/**
+ * A class responsible of parsing the GQL and CommunityOracle data into the OutputData file
+ */
+export class DailyOutput {
+	private data = this.read();
+	private latestBlock = 0;
+	private latestTimestamp = 0;
+	private bundlesTips: { [address: string]: number } = {};
+
+	/**
+	 * Takes the data from the previously generated data, fallbacking to the base template if not present
+	 * @throws if the validation of the output file fails
+	 * @retuns {OutputData}
```

Comment:
spelling nit: 

```suggestion
	 * @returns {OutputData}
```

Although usually I prefer to let the TypeScript declarations describe return and parameter information, unless there is something particularly important I want to convey in the docblock about a variable

Reply:
e7564f47333dc4acbcb482e024d5cfe4adef554d

Reply:
I'll have your suggestion in mind for future PRs. Thanks!